WordPress/Classes/ViewRelated/Jetpack/Install/Webview/JetpackConnectionWebViewController.swift

```diff
@@ -68,11 +70,18 @@ class JetpackConnectionWebViewController: UIViewController {
 
         let request = URLRequest(url: url)
         webView.load(request)
+
+        WPAnalytics.track(.installJetpackWebviewSelect)
     }
 
     @objc func cancel() {
         delegate?.jetpackConnectionCanceled()
     }
+
+    private enum AnalyticsErrorState {
```

Comment:
why an `enum` here over a `bool`?

Reply:
It's something I always wanted to try. The main reason is because enumerations are explicit and descriptive. Having the cases for _tracked_ and _non tracked_ rather than a switch on a Bool value. This probably is better described [here](http://narf.pl/posts/2cased).
What do you think?

Reply:
imo `analyticsErrorState` is an ambiguous name, so understanding the code requires keeping more in mind, whereas something like `analyticsErrorWasTracked` can be understood anywhere it's encountered. So my main discomfort is the variable name, really. However it maps very well to a boolean so that's my preference.

Fwiw, I like that article's suggestion for uses like `move.dynamic = .gentle/.rapid`, because the two states don't map well to true/false: if it were a boolean called `move.isRapid` there is ambiguity about what the `false` state means. Is "not fast" gentle, or maybe it's periodic, or hyperspeed?